dotCMS/src/main/java/com/dotmarketing/services/HostServices.java

```diff
@@ -70,27 +70,31 @@ public static InputStream buildStream(Host host, boolean EDIT_MODE) throws DotDa
 			
 			 }
 		}
+        java.io.BufferedOutputStream tmpOut = null;
+        OutputStreamWriter out = null;
 		try {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change this "try" to a try-with-resources. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2093)

Reply:
not valid